### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by undefobj**
Still needs images

### CODE REVIEW

File: cli/multienv.md [hunk 1]
```diff
@@ -1,15 +1,47 @@
 # Multiple environments and team workflows (beta)
 
-In this section, we'll go over how you can manage multiple environments of your Amplify project (backend + frontend) and managing this project within a team or outside a team using the Amplify CLI & Git. 
-This feature is still work in progress and you would have to install a beta version of the CLI to check out all the features mentioned in this section.
+This section outlines how you can manage multiple environments of your Amplify project (backend + frontend) as well as using a project within a team or outside a team using the Amplify CLI & Git. 
+This functionality is still work in progress and you would have to install a beta version of the CLI to check out all the features mentioned in this section.
+
+**Note**: We recommend backing up any existing Amplify projects first before migrating to the new version.
```
Comment (kaustavghosh06) [inline cli/multienv.md]: Do we want to add this right now? Let's add notes about migration on Tuesday. Or we can have a "Coming soon:" section?

File: cli/multienv.md [hunk 2]
```diff
@@ -1,15 +1,47 @@
...
+
 Install the CLI using the following command:
 ```
 npm install -g @aws-amplify/cli@multienv
 ```
+
+## Concepts
+
+Amplify fits into the standard Git workflow where you switch between different branches using the `env` command. Similarly to how you will run `git checkout -b BRANCHNAME` you will run `amplify env checkout ENVIRONMENT_NAME`. The below diagram shows a workflow of this.
```
Comment (kaustavghosh06) [inline cli/multienv.md]: `git checkout BRANCHNAME`.
 I believe `-b` is used only when you want to create a new branch

File: cli/multienv.md (hunk 1 above)
Comment (undefobj) [reply cli/multienv.md]: I like the idea of a Coming soon, that way people know. I more put this in here so customers don't break in the meanwhile.

File: cli/multienv.md (hunk 2 above)
Comment (undefobj) [reply cli/multienv.md]: Yeah I didn't want to go into all those details, should we call it out?

**Pull request merged**